### ISSUE TITLE
Clickable nicknames in messages like in dgg

### ISSUE THREAD

**Issue by vyneer** [vyneer/orvods-go#9]
Likely possible, but would be 100% optional.

**Comment by gabrieljablonski** [comment]
Have you thought about just ripping the dgg chat gui source code and feeding the logs through the websocket?
Seems like it would be easier than to just reimplement these features you have listed in the issues.

**Comment by vyneer** [comment]
No, never thought about that, but that seems like a pretty good idea!
I'll work on it when I have the time.

Thanks ❤

**Comment by gabrieljablonski** [comment]
No problem. I'm a fan of the project so I take a look from time to time for little things to contribute, so make sure to keep the issues up to date with where you're at :)
